lib/samples/show_realistic_light_and_shadows/show_realistic_light_and_shadows.dart

```diff
@@ -0,0 +1,198 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
```

Comment:
```suggestion
```

Reply:
I need to use the package to format the dateTime.
It is overridden in the sample `pubspec.yaml`, not appear in the dependency section. that is why there is a warning.

Reply:
Ah I see. I think we should be aiming for no warnings. Eventually we'll want a PR check to enforce that too. Can we add this as a dependency in pubspec.yaml, at the required min version, instead of just the override if it's actually required for this sample? I think that would be cleaner.